spec/API_specification/searching_functions.md

```diff
@@ -86,6 +91,11 @@ Returns the indices of the array elements which are non-zero.
 (function-where)=
 ### where(condition, x1, x2, /)
 
+```{note}
+
+This function does not support boolean array indexing.
```

Comment:
Maybe I am missing something, but the shape of the output is determined according to broadcasting rules applied to `condition`, `x1`, and `x2`. The output values are data-dependent, but I don't think that the output shape is data-dependent.

Reply:
Given the discussion in #84, it seems that JAX has this issue and Ralf also pointed out that `where` may be problematic.